code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -123,7 +123,7 @@
 
 /obj/item/fugu_gland/afterattack(atom/target, mob/user, proximity_flag)
 	. = ..()
-	if(proximity_flag && isanimal(target))
+	if(proximity_flag && isanimal(target) && !isguardian(target))
```

Comment:
Why not just use the `banned_mobs` list fugu glands have instead of doing this?

Reply:
fair assesment i thought guardians were a branched type of things like they use to be with dexterous

Reply:
fixed in latest commit